buddyboss/buddyboss-platform
# Fixed the issue with same checkboxes and radio button in member searc…

…h form

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1903 .

### How to test the changes in this Pull Request:

1. Go to Profile Search setting.
2. Add two radio buttons in form fields.
3. Scroll down to member page and try clicking those radio buttons.
4. See error

### Proof Screenshots or Video

https://www.loom.com/share/c8d4302abfec4d64ad9a5492a596e5c9

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
